djstripe/models.py

```diff
@@ -1626,6 +1643,10 @@ class Card(StripeSource):
         help_text="If the card number is tokenized, this is the method that was used."
     )
 
+    customer = models.ForeignKey(
+        "Customer", on_delete=models.CASCADE, related_name="sources"
```

Comment:
This is going to cause issues later when other payment sources also want to use the same `related_name`. This works for now, for backwards compatibility, but just wanted to note it. We may want to change how we do this sooner rather than later, then deprecate it in the next release, and then remove it (or move it to `PaymentMethod` instead).